tests/test_discretisation/test_check_estimator_discretisers.py

```diff
@@ -7,14 +7,16 @@
     DecisionTreeDiscretiser,
     EqualFrequencyDiscretiser,
     EqualWidthDiscretiser,
+    TargetMeanDiscretiser,
 )
 from tests.estimator_checks.estimator_checks import check_feature_engine_estimator
 
 _estimators = [
     DecisionTreeDiscretiser(regression=False),
     EqualFrequencyDiscretiser(),
     EqualWidthDiscretiser(),
-    ArbitraryDiscretiser(binning_dict={"0": [-np.Inf, 0, np.Inf]}),
+    ArbitraryDiscretiser(binning_dict={"0": [-np.Inf, 0, np.Inf]})
```

Comment:
, is missing here

Reply:
Are we now including the ArbitraryDiscretiser?

Reply:
Nope. This is a common test script that runs common tests on all discretisers.

Reply:
there is a , missing after `ArbitraryDiscretiser(binning_dict={"0": [-np.Inf, 0, np.Inf]})` this is probably causing many tests to fail,